Server/VikingVault.Services/ExchangeService.cs

```diff
@@ -38,12 +38,13 @@ public List<BankAccount> Exchange(string token, UpdateBankAccountModel bankAccou
 
             Transaction transaction = new Transaction
             {
-                User = returnedUser,
+                Sender = returnedUser,
                 Type = "Exchange",
-                Currency = bankAccountSell.CurrencyType,
+                Currency = bankAccountBuy.CurrencyType,
                 Date = DateTime.Now,
                 Amount = exchangeInfo.Amount,
-                OtherParty = exchangeInfo.CurrencyType
+                Receiver = returnedUser,
+                Details = exchangeInfo.CurrencyType
```

Comment:
Why is this called `Details` when it stores the an `CurrencyType` ?

Reply:
Because exchangeInfo.currencyType actually stores details but it uses the same model as bankAccountSell